### ISSUE TITLE
Processed coordinates should have appropriate decimal precision

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/biocache-store#85]
A support email highlighted the lat/lng values of some records being precise down to nanometre scale. His suggestion was no more than 6 decimal places:

> On 21 Sep 2015, at 6:18 pm, Bob Mesibov <robert.mesibov@gmail.com> wrote:
But you know that it's impossible to have nanometre accuracy in geographic coordinates, however they are originally obtained before conversion. Please consider the (spurious) accuracy of a Google Earth screen read-out in degree-minutes-seconds, e.g. 41°10'16.55"S 144°41'30.50"E. That's the best implied accuracy I've seen in any published biological records, and it implies the location is known to the nearest 0.01 second. One second of latitude is about 31m, so 0.01 of a second is 0.31m. One degree of latitude is ca 111km, so 0.000001 degree is 0.11m. Why would you ever have decimal degree figures in ALA with better than 0.000001 implied accuracy?

Example record: http://biocache.ala.org.au/occurrences/f4d2e399-f269-4852-86f4-6fc8dbca9262

`"Latitude":"-37.73736070058887","Longitude":"144.87507820129395"`

Suggestion is to process the lat & lng to be no more than 6 decimal places and less if there are supporting coordinate precision or coordinate uncertainty values(?).

**Comment by adam-collins** [comment]
Related to #28.

**Comment by djtfmartin** [comment]
I'd be interested in Lee's thoughts on this. I agree with the suggestion but we've deliberately avoided doing anything with raw coordinate values in the past where we dont need to (i.e. sensitive species or reprojections)

**Comment by Tasilee** [comment]
I agree with Bob that we should not imply a higher coordinate precision than the user provided. If coordinates are translated to decimal degrees, ideally we should reflect the original precision. verbatumCoordinates are available for reference.

**Comment by djtfmartin** [comment]
In Nick's example, 

http://biocache.ala.org.au/occurrences/f4d2e399-f269-4852-86f4-6fc8dbca9262

there is no translation of the coordinates. There is also no metadata indicating how the coordinates where derived.

Should we reduce the number of decimal places as I think Bob is suggesting (from 15 to 7) ?

**Comment by Tasilee** [comment]
Given even 'standard' (vs differential) GPS accuracy, I'd agree with 6 decimal places. This is better than 1m on the ground.
